src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs

```diff
@@ -652,11 +652,10 @@ internal static bool IsPackageReferenceProject(Project project)
         /// <param name="userInputFrameworks">A list of frameworks</param>
         /// <param name="assetsFile">Assets file for all targets and libraries</param>
         /// <param name="transitive">Include transitive packages/projects in the result</param>
-        /// <param name="includeProjects">Include project references in top-level and transitive package lists</param>
```

Comment:
So in some places, we seem to want to include projects only in the default `dotnet list package`, but in other places, we seem to filter them out. 

The spec did not clearly call out the whole `dotnet list package` lists projects, but `--outdated`, `--vulnerable` or `--deprecated` does not. 

But maybe there is something to be discussed here?

Reply:
Sorry, I don't understand why we ever want to include projects in `dotnet list package`. There's already a `dotnet list reference` command to show project references.

Reply:
🤷 The feature design does not explicitly call out either one. The implementation tries to do both. 

I'm ok with listing just the packages, given the list reference command is available, but it just made me wonder whether anyone might truly depend on this.